scripts/make/generate_notice.py

```diff
@@ -0,0 +1,150 @@
+from __future__ import print_function
+
+import glob
+import os
+import datetime
+import argparse
+import json
+import csv
+import re
+import pdb
+import copy
+import subprocess
+import fnmatch
+import textwrap
+import sys
+import tempfile
+
+DEFAULT_BUILD_TAGS = "darwin,linux,windows"
```

Comment:
do we need windows here?

Reply:
Prob not, but this script is required and i'd prefer to avoid changes.

Reply:
alrighty